cranelift/filetests/src/runner.rs

```diff
@@ -13,7 +13,7 @@ use std::path::{Path, PathBuf};
 use std::time;
 
 /// Timeout in seconds when we're not making progress.
-const TIMEOUT_PANIC: usize = 10;
+const TIMEOUT_PANIC: usize = 60;
```

Comment:
Oh sorry I meant to ask earlier, but what's this doing?

Reply:
Oh this is just bumping the time limit for each individual test. If I run with `RUST_LOG=trace` then I was running into the time limit for run tests.